File: src/main/java/my/maryvk/maryvkweb/service/VkService.java
package my.maryvk.maryvkweb.service;

import my.maryvk.maryvkweb.domain.RelationType;
import my.maryvk.maryvkweb.domain.User;

import java.util.List;

public interface VkService {
    List<Integer> getConnectedIds(Integer userId, RelationType relationType);
    User getUser(Integer id);
}

File: src/main/java/my/maryvk/maryvkweb/service/VkService.kt
package my.maryvk.maryvkweb.service

import my.maryvk.maryvkweb.domain.RelationType
import my.maryvk.maryvkweb.domain.User

interface VkService {
    fun getConnectedIds(userId: Int, relationType: RelationType): List<Int>?
    fun getUser(id: Int): User
}